Modules/Relationships/test/Relationships.Domain.Tests/Tests/Aggregates/Relationships/RelationshipReactivationRequestTests.cs

```diff
@@ -7,30 +7,30 @@
 using static Backbone.Modules.Relationships.Domain.Tests.TestHelpers.TestData;
 
 namespace Backbone.Modules.Relationships.Domain.Tests.Tests.Aggregates.Relationships;
-public class ReactivateRelationshipRequestTests
+public class RelationshipReactivationRequestTests
 {
     [Fact]
-    public void Requesting_relationship_reactivation_transitions_relationship_to_status_reactivation_requested()
+    public void Relationship_reactivation_request_does_not_change_relationship_status()
```

Comment:
It's rarely a good idea to test that something did not happen. Because: where do you stop? You don't want to check every single property of an object.

Reply:
Changes made.